components/Cards/examples/CardExampleViewController.swift

```diff
@@ -24,6 +24,13 @@ class CardExampleViewController: UIViewController {
   var colorScheme = MDCSemanticColorScheme()
   var shapeScheme = MDCShapeScheme()
   var typographyScheme = MDCTypographyScheme()
+  var containerScheme: MDCContainerScheme {
```

Comment:
Not sure this is the right way to go. The convention in our catalog is to use an injected instance property that's initialized in viewDidLoad if nil. if containerScheme is a computed property, then it can't be injected.

Reply:
We’re in a strange middle state where the app itself doesn’t have the container scheme yet, but our examples need to use container schemes. When we implement the app’s container scheme and corresponding injection logic we’ll need to update these examples accordingly.